src/css/index.scss

```diff
@@ -0,0 +1,9 @@
+*,
+*:before,
```

Comment:
I've never seen these *before and *after things in CSS before - can you add a screenshot of what this looks like?

Reply:
```:before , :after``` are <a href="https://css-tricks.com/almanac/selectors/a/after-and-before/">pseudo element selectors</a>  what I am saying here is that for all elements on the page I want to use box-sizing border box and for all child elements I want them to inherit this rule.  
here is the layout as is.  I do want to change the class of grid to use grid-template-column.
![screencapture-localhost-8081-2019-10-21-09_04_59](https://user-images.githubusercontent.com/45470456/67224216-f1bced00-f3ed-11e9-9e31-9dc6dc6d3ce1.png)